_data/sites.json

```
@@ -5041,6 +5032,17 @@
         ]
     },
 
+    {
+        "name": "Neon Pagamentos",
+        "url": "http://suporte.neon.com.br/hc/pt-br/articles/360019117253-Quero-cancelar-minha-conta-Como-fa%C3%A7o",
```

Can the commit adding this entry be squashed with the commit adjusting the URL encoding? I'd prefer those changes both be incorporated in a single commit.

---

Also, it looks like the URL should be the following:
```suggestion
        "url": "https://suporte.neon.com.br/hc/pt-br/articles/360019117253-Quero-cancelar-minha-conta-Como-fa%C3%A7o",
```